webpack.config.js

```diff
@@ -68,6 +68,17 @@ if (TARGET !== 'readme') {
         ).filter((x, i, a) => a.indexOf(x) === i);
     }
 
+    function getPOFilenames(_path, extension) {
+        return fs
+            .readdirSync(_path)
+            .filter(
+                item =>
+                    fs.statSync(path.join(_path, item)).isFile() &&
+                    (extension === undefined || path.extname(item) === extension)
```

Comment:
I see that `extension` is defined in all invocations. Is there a reason to allow it to be unspecified, still?

Reply:
I wanted to make sure that it could be reusable and would load only what I asked